src/client/datascience/export/exportDialog.ts

```diff
@@ -0,0 +1,70 @@
+import { inject, injectable } from 'inversify';
+import * as path from 'path';
+import { SaveDialogOptions, Uri } from 'vscode';
+import { IApplicationShell, IWorkspaceService } from '../../common/application/types';
+import * as localize from '../../common/utils/localize';
+import { computeWorkingDirectory } from '../jupyter/jupyterUtils';
+import { ExportFormat, IExportDialog } from './types';
+
+// File extensions for each export method
+export const PDFExtensions = { PDF: ['pdf'] };
+export const HTMLExtensions = { HTML: ['html', 'htm'] };
+export const PythonExtensions = { Python: ['py'] };
+
+@injectable()
+export class ExportDialog implements IExportDialog {
```

Comment:
This file used to be called ExportFileManager. It really just puts up the dialog now.

Reply:
Bleh, github didn't pick up the rename though? Can't easily tell what changed...